packages/actor-init-sparql/lib/ActorInitSparql-browser.ts

```diff
@@ -169,6 +169,32 @@ export class ActorInitSparql extends ActorInit implements IActorInitSparqlArgs {
     // Execute query
     const resolve: IActionQueryOperation = { context, operation };
     const output = await this.mediatorQueryOperation.mediate(resolve);
+    // Set bindings
+    if ((<any> output).bindingsStream) {
```

Comment:
Casting to `any` should be avoided if possible. Does the compiler accept `'bindingsStream' in output` instead?